invenio_app_rdm/requests_ui/templates/semantic-ui/invenio_requests/community-submission/index.html

```diff
@@ -64,9 +63,8 @@
         >
           {{ _("Record") }}
         </a>
-      </div>
-    {% endif %}
-
+      {% endif %}
+    </div>
```

Comment:
All of this fixes the absence of “Conversation” tab on the request page (in case if request is closed and the viewer is not the owner of the request - we don't show the record tab. But we still should show the conversation tab (?right?))

Before:
![Screenshot 2024-07-25 at 14 02 27](https://github.com/user-attachments/assets/91ddc8cf-a464-4999-bf2b-001b26f2bdb2)

After:
![Screenshot 2024-07-25 at 14 02 04](https://github.com/user-attachments/assets/13a55eec-0589-494a-bd0d-7cc96f292dd9)